components/pedigree/resources/src/main/resources/pedigree/model/import.js

```diff
@@ -705,6 +705,7 @@ define([
      *                            First a match versus an existing ID is checked, if not found a check against "externalId",
      *                            if not found a check against "name" and finally "firstName".
      *                            If one of the parents is given and the other one is not a virtual new node is created
+     *   - "phenotipsId": The id of the PhenoTips document this node is linked to (default: none)
```

Comment:
good point, but not sure what is better. DocumentId may give ideas that you can link to some other documents, which wont work correctly (since this link only works correctly in pedigree when it links to a specifically PT document)